challenges/5-arraycardio/arraycardio.js

```diff
@@ -121,7 +173,25 @@ function copy(arr) {
         console.log(newArr) --> [1,2,9]
 */
 function insertIntoNewArray(arr, index, element) {
-    // implementation
+    let newArr = [];
+    for (let i = 0; i < arr.length; i++) {
+        newArr.push(arr[i]);
+    }
+    newArr.push(element);
+    if (newArr[index] != element) {
+        let i = newArr.length - 1;
+        let placeholder = newArr[i];
+        newArr[i] = newArr[i - 1];
+        newArr[i - 1] = placeholder;
+    }
+    /*for (let n = 1; newArr[index] == element; n++) {
+      let i = newArr.length - 1; 
+      let placeholder = newArr[i];
+      newArr[i] = newArr[i-1]; 
+      newArr[i-1] = placeholder;
+    }*/
+
```

Comment:
I think I wrote the tests wrong. Swap will not work in this case.
What is returned when I do `insertIntoNewArray([9,2], 0, 3)`?